nx/lib/nx.ex

```diff
@@ -4208,8 +4208,11 @@ defmodule Nx do
   precision to make sure the sum does not overflow:
 
       iex> Nx.sum(Nx.tensor([[101, 102], [103, 104]], type: {:u, 8}, names: [:x, :y]))
+      ** (RuntimeError) Torchx does not support unsigned 64 bit integer (explicitly cast the input tensor to a signed integer before taking sum)
```

Comment:
If we change this like this, the `Nx` suite will fail. My suggestion: remove this doctest and add it to test/nx_test.exs instead.